src/views/Orders.vue

```diff
@@ -455,6 +455,23 @@ export default defineComponent({
         });
       return jobAlert.present();
     },
+    fetchJobs(){
+      this.store.dispatch("job/fetchJobs", {
+        "inputFields": {
+          "enumTypeId": "ORDER_SYS_JOB"
+        }
+      });
+      this.store.dispatch("job/fetchJobs", {
```

Comment:
I think batch jobs should have enumTypeId as ORDER_SYS_JOB and we are already fetching those jobs

Reply:
Sir, the More Jobs section is not visible if we remove the code.